src/playbacks/html5_video/html5_video.js

```diff
@@ -95,6 +95,7 @@ export default class HTML5Video extends Playback {
     super(...args)
     this._destroyed = false
     this._loadStarted = false
+    this._bufferingState = false
```

Comment:
I'd change this to `this._isBuffering`, or something like that. When talking about the `State` suffix, I understand it's something with specific values instead of a simple boolean flag.